src/inverseRobot.js

```diff
@@ -8,6 +8,20 @@
 
 function inverseRobot(robot) {
   // write code here
+  const fixRobot = {};
+
+  for (const key in robot) {
+    const fixKey = robot[key];
+    const fixValue = key;
```

Comment:
The variable names fixKey and fixValue can be made more descriptive, like swappedKey